api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -142,52 +138,33 @@ private String convertClusterName(String workspaceId) {
 
   @Override
   public ResponseEntity<Void> localizeNotebook(String workspaceNamespace, String workspaceId, List<FileDetail> fileList) {
-    try{
+    try {
       String clusterName = convertClusterName(workspaceId);
-      this.notebooksService.localize(workspaceNamespace, clusterName, convertfileDetailsToJson(workspaceNamespace, workspaceId, fileList));
-      } catch (ApiException e) {
-        throw new RuntimeException(e);
-      }catch(Exception e) {
+      this.notebooksService.localize(workspaceNamespace, clusterName, convertfileDetailsToMap(workspaceNamespace, workspaceId, fileList));
+    } catch (ApiException e) {
         throw new RuntimeException(e);
     }
-    return null;
+    return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
   }
 
   /**
-   * Create a map with key as ~/workspaceName/workspaceID/fileName and value as the gs://firecloudBucket name/filename
+   * Create a map with key as ~/workspaceName/workspaceID/fileName and value as file path which should be in format the gs://firecloudBucket name/filename
```

Comment:
Does this actually copy files to the directory ~/workspaceName/workspaceID on the VM?

If so, why that directory? The workspace is already implicit in the scope of the VM... we don't necessarily have to have it encoded in a directory name.

Is there any expectation of where notebook files live? Is there going to be a notebooks directory somewhere?)

Reply:
it does create two directories : workspace name -> workspace Id and then copy the file into that. Wasnt too sure about the norm.Can we all discuss the folder structure both for google bucket (input) as well as this so that everyone is on the same page?

Reply:
Yeah. Will start a thread on Slack about this.